### ISSUE TITLE
[FYI] How to Transfer a Repo

### ISSUE THREAD

**Issue by 274188A** [vanderbilt-redcap/twoColumnChecks#6]
I read in he comments previously that @scottjpearson  was open to transferring the Repo
Link below describes the steps:
https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository
(ps i'm not volunteering to be the new owner! :-)
HTH

**Comment by mmcev106** [comment]
@lsgs, are you interested in adopting this and making your fork the primary?

**Comment by lsgs** [comment]
Yeah, I guess so @mmcev106. I don't imagine it'll be too much trouble and maybe it can be properly deprecated when the new and improved one coumes out (see https://community.projectredcap.org/questions/57864/two-column-checks-external-module.html).

**Comment by mmcev106** [comment]
@lsgs, it looks like github might require you to remove your fork before it will let me transfer the repo to you.  If you want to go ahead and remove it, I will try the transfer again.
